tests/acceptance/table-rerenders-when-rows-change-test.js

```diff
@@ -1,24 +1,15 @@
 import Ember from 'ember';
-import { module, test } from 'qunit';
-import startApp from '../../tests/helpers/start-app';
-import $ from 'jquery';
-
-module('Acceptance | table rerenders when rows change', {
-  beforeEach() {
-    this.application = startApp();
-  },
-
-  afterEach() {
-    Ember.run(this.application, 'destroy');
-  }
-});
+import { test } from 'qunit';
+import moduleForAcceptance from '../../tests/helpers/module-for-acceptance';
+
+moduleForAcceptance('Acceptance | table rerenders when rows change');
 
 test('basic table properly re-renders when rows are removed', function(assert) {
   visit('/examples/basic-table');
 
   andThen(() => {
-    let rows = $('.table-columns tr').length;
-    let headers = $('.table-columns th').text().trim();
+    let rows = Ember.$('.table-columns tr').length;
+    let headers = Ember.$('.table-columns th').text().trim();
```

Comment:
Thoughts on changing `import jQuery from 'jquery'` above vs updating this to `Ember.$`? If you're thinking it's more clear that `Ember.$` does not reference the window scope, I'm good with that.